docs/tutorials/processing/dask.md

```diff
@@ -171,9 +171,51 @@ channels = [imread(file_pattern.format(i)) for i in range(nchannels)]
 stack = da.stack(channels)
 stack.shape  # (2, 600, 64, 256, 280)
 stack[0, 0].compute()  # incurs a single file read
+```
+
+## A real example: high-resolution human brain imaging dataset
+
+In this example, we are going to view and process a high-resolution human brain
+imaging dataset. The dataset was obtained from [Dr. Brian Edlow's Lab](https://www.comarecoverylab.org/)
+for NeuroImaging of Coma and Consciousness, which is dedicated to promoting
+recovery of consciousness in people with severe brain injuries. This dataset is
+a 100 micron resolution magnetic resonance imaging (MRI) scan of an ex vivo
+human brain specimen. The brain specimen was donated by a [58-year-old woman](https://www.youtube.com/watch?v=Q-9jzBkoNuI) who had no history of neurological disease and died of non-neurological causes.
+This dataset was originally described in [Edlow, B.L., Mareyam, A., Horn, A. _et al._ 7 Tesla MRI of the ex vivo human brain at 100 micron resolution. Sci Data 6, 244 (2019)](https://doi.org/10.1038/s41597-019-0254-8).
+
+```{admonition} Getting the data
+
+You can download the dataset from [here](https://datadryad.org/stash/dataset/doi:10.5061/dryad.119f80q).
+A `.zip` file will be downloaded, and after extracting the compressed dataset
```

Comment:
```suggestion
In this example we use the [FLASH25_100um_TIFF_Axial data](https://datadryad.org/stash/downloads/file_stream/182482). This link will download a `.zip` file  and after extracting the compressed dat set,
```

Reply:
Is it possible to add the expected size of the download to this sentence as well? Maybe in brackets after the link or something. I think that's helpful for users to know whether or not they have bandwidth for it or not.

Reply:
Previously the unzipped size was given, but I've now added the zipped size as well (~2 Gb)